test/langtools/jdk/javadoc/doclet/testNewLanguageFeatures/TestNewLanguageFeatures.java

```diff
@@ -464,7 +464,7 @@ class in pkg2">ParamTest</a><wbr>&lt;<a href="../Foo.html" title="class in pkg2\
         //=================================
         checkOutput("index-all.html", true,
                 """
-                    <a href="pkg2/Foo.html#method(java.util.Vector)" class="member-name-link">method(Vector&lt;Object&gt;)</a>"""
```

Comment:
Is this caused by replacing `Util.flatSignature` with `makeSignature(.., false, true)`?

Reply:
Yes.